packages/app/src/app/components/Preview/DevTools/Terminal/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Terminal } from 'xterm';
+
+export type TerminalWithFit = Terminal & {
+  fit(): void;
```

Comment:
@CompuIves It's more commonly used to just use the function name and put the type after the colon (`:`).

```suggestion
  fit: () => void;
```

Reply:
Hmm, in this case the function isn't an arrow function though. I think it's better to keep it like this.

Reply:
Just to be consistent with the rest of the types in the codebase